microovn/api/services/control.go

```diff
@@ -36,6 +38,16 @@ func enableService(s state.State, r *http.Request) response.Response {
 	if !node.CheckValidService(requestedService) {
 		return response.InternalError(errors.New("Service does not exist"))
 	}
+	if node.SrvName(requestedService) == node.SrvCentral {
+		shutdownCtx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+		err = ovn.Refresh(shutdownCtx, r.Context(), s)
```

Comment:
This appears to be a nice workaround for the fact that we currently have no mechanism to trigger cluster wide refresh on changes to the central DBs!